### PR TITLE
Feature/candidate total filters

### PR DESCRIPTION
**Pull request by jmcarp**
* Add full-text search on candidate totals
* Fix financial range filters on candidate totals

### CODE REVIEW

File: webservices/args.py [hunk 1]
```diff
@@ -449,14 +450,14 @@ def _deserialize(self, value, attr, data):
...
-    'min_debts_owed_by_committee': fields.Int('Minimum debt'),
-    'max_debts_owed_by_committee': fields.Int('Maximum debt'),
+    'min_receipts': Currency(description='Minimum aggregated receipts'),
+    'max_receipts': Currency(description='Maximum aggregated receipts'),
+    'min_disbursements': Currency(description='Minimum aggregated disbursements'),
+    'max_disbursements': Currency(description='Maximum aggregated disbursements'),
+    'min_cash_on_hand_end_period': Currency(description='Minimum cash on hand'),
+    'max_cash_on_hand_end_period': Currency(description='Maximum cash on hand'),
+    'min_debts_owed_by_committee': Currency(description='Minimum debt'),
+    'max_debts_owed_by_committee': Currency(description='Maximum debt'),
```
Comment (LindsayYoung) [inline webservices/args.py]: If currency is a better type to use we might want to make the other max/min fields currency as well.

Comment (jmcarp) [inline webservices/args.py]: Updated range fields for the incomplete `totals_committee_aggregate` args below.

**Comment by LindsayYoung** [comment]
Needs district filter for 18F/openFEC-web-app/pull/1073

**Comment by jmcarp** [comment]
Added district filter.

**Pull request merged**